chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -394,9 +393,16 @@
                     <key>frequency</key>
                     <value>WEEKLY</value>
                 </entry>
+                <entry>
+                    <key>range</key>
+                    <value>7</value>
+                </entry>
+                <entry>
+                    <key>parameters</key>
+                    <value>[{"label": "Range (in days)", "key": "range", "type": "int", "min": "7"} ]</value>
```

Comment:
No specific change request, but why have a min but no max? I thought the ticket talked about a max value?

Reply:
I think the range should be 1-365.  Updated the code to reflect this.